taskcluster/requirements.txt

```diff
@@ -1,8 +1,8 @@
 #
-# This file is autogenerated by pip-compile with python 3.6
+# This file is autogenerated by pip-compile with python 3.7
```

Comment:
👀  I'm assuming this is tested and intentional

Reply:
mozillareleases/taskgraph:decision-mobile-35a3e77038af77191b9eb65d822a2ee34839e44d176859a69c5086f60101947c (as referenced in .tc.yml) has python 3.8.10, so this should be ok.

Reply:
Taskgraph also doesn't even run with 3.6 anymore, it has some 3.7+ only syntax :)